recipes/arb-bio/6.1.rev16223/meta.yaml

```diff
@@ -0,0 +1,93 @@
+{% set name = "ARB" %}
+{% set version = "6.1.rev16223" %}
+
+package:
+  name: {{ name|lower }}-bio
+  version: {{ version }}
+
+source:
+  svn_url: http://vc.arb-home.de/readonly/trunk
```

Comment:
I had pondered creating a tar ball. The ones at ftp.arb-silva.de/ARB/builds have seized to appear, but I wasn't sure about self-hosting this as it wouldn't create a safer spot.

How do I use cargo port? We could use this Trac URL to pull the revision as zip archive:
http://bugs.arb-home.de/browser/trunk?rev=16228&format=zip